### ISSUE TITLE
Has SLA always false

### ISSUE THREAD

**Issue by quki** [azkaban/azkaban#2434]
## My Azkaban info
- azkaban version:  3.71.1 ~
- mode: multi executors (1 web server - 1 executor - mysql DB)

## Issue 
**In web UI [Scheduling] tab, 'Has SLA' always false even though set SLA options correctly.**

![image](https://user-images.githubusercontent.com/12330436/73432981-3231ec80-4387-11ea-9f0c-bce1e3b718cd.png)

**Comment by djaiswal83** [comment]
Thanks for brining this up. Yes, we are aware of this and intend to fix it in the future.

**Comment by ismailsimsek** [comment]
is this fixed with #2506 ?

**Comment by djaiswal83** [comment]
No, #2506 is a correctness fix. @ypadron-in may have more context on it.
